index.html

```diff
@@ -77,99 +127,7 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span>I'm a Software
         <div class="action-btn"><img src="img/Disabled.png" alt=""></div>
         <h2 class="headline"> My Recent Works </h2>
     </section>
```

Comment:
- [ ] **Required** You are required to have a couple of projects in your website, that contain buttons to fire the popups when clicked, as the website is at the moment we do not have such projects. kindly ensure the projects are shown.
![image](https://user-images.githubusercontent.com/27709832/146393802-49979450-370e-4e4a-9d98-d576fb001f31.png)
- [ ] **Required** The popups should also have X as a button to be clicked to close them.
- [ ] **Required** Finally, before requesting the next review, kindly go through the project requirement to ensure they are all met.